ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,225 @@
+    @Published public var iconSize: CGFloat!
+    @Published public var largeIconSize: CGFloat!
+    @Published public var subtitleFont: UIFont!
+    @Published public var textFont: UIFont!
+
+    var iconStyle: MSFListIconVnextStyle!
+    var layoutType: MSFListCellVnextLayoutType!
+
+    public init(layoutType: MSFListCellVnextLayoutType, iconStyle: MSFListIconVnextStyle) {
+        self.layoutType = layoutType
+        self.iconStyle = iconStyle
+        self.themeAware = true
+
+        didChangeAppearanceProxy()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        let appearanceProxy: ApperanceProxyType
+
+        if iconStyle == MSFListIconVnextStyle.iconOnly {
+            appearanceProxy = StylesheetManager.S.IconOnlyListTokens
+        } else {
+            appearanceProxy = StylesheetManager.S.MSFListTokens
+        }
+
+        backgroundColor = appearanceProxy.backgroundColor.rest
+        borderColor = appearanceProxy.borderColor
+        disclosureIconForegroundColor = appearanceProxy.disclosureIconForegroundColor
+        iconColor = appearanceProxy.iconColor
+        leadingTextColor = appearanceProxy.labelColor
+        subtitleColor = appearanceProxy.sublabelColor
+        trailingItemForegroundColor = appearanceProxy.trailingItemForegroundColor
+
+        highlightedBackgroundColor = appearanceProxy.backgroundColor.pressed
+
+        borderSize = appearanceProxy.borderSize
+        disclosureInterspace = appearanceProxy.disclosureInterspace
+        disclosureSize = appearanceProxy.disclosureSize
+        horizontalCellPadding = appearanceProxy.horizontalCellPadding
+        iconInterspace = appearanceProxy.iconInterspace
+        iconSize = appearanceProxy.iconSize.icon
+        largeIconSize = appearanceProxy.iconSize.largeIcon
+        subtitleFont = appearanceProxy.textFont.subLabel
+        textFont = appearanceProxy.textFont.label
+    }
+}
+
+public struct MSFListView: View {
+    var cells: [MSFListVnextCell]
+    @ObservedObject var state: MSFListVnextState
+    @ObservedObject var tokens: MSFListTokens
+
+    public init(cells: [MSFListVnextCell],
+                layoutType: MSFListCellVnextLayoutType,
+                iconStyle: MSFListIconVnextStyle) {
+        self.cells = cells
+        self.state = MSFListVnextState()
+        self.tokens = MSFListTokens(layoutType: layoutType, iconStyle: iconStyle)
+
+    }
+
+    public var body: some View {
+        List {
+            ForEach(cells, id: \.self) { item in
+                MSFListCellView(cell: item, tokens: tokens)
+                    .onTapGesture(perform: item.handler ?? {})
+            }
+            .border(state.hasBorder ? Color(tokens.borderColor) : Color.clear, width: state.hasBorder ? tokens.borderSize : 0)
+            .frame(minHeight: tokens.layoutType.height)
+            .listRowInsets(EdgeInsets())
+        }
+        .frame(width: UIScreen.main.bounds.width, height: 300)
+        .environment(\.defaultMinListRowHeight, 0)
```

Comment:
What is using this environment value on the List hierarchy?

Reply:
SwiftUI's List currently has built-in default padding in between the rows, and there is no modifier to remove them without hardcoding negative padding. This helps with eliminating the extra padding so that the tokens can be reflected more accurately